ddtrace/ext/__init__.py

```diff
@@ -1,7 +1,4 @@
-from enum import Enum
```

Comment:
if we no longer need enum, we can also avoid having to install the backport

Reply:
Looks like it's still used in `ddtrace.ext.test` but perhaps we could do away with that? I'll have a look

Reply:
I think `ddtrace.ext.test.Status` could also follow the same change, since it's effectively the same problem as `SpanTypes` as before.